system/Entity.php

```diff
@@ -124,18 +124,7 @@ public function fill(array $data = null)
 
 		foreach ($data as $key => $value)
 		{
-			$key = $this->mapProperty($key);
-
-			$method = 'set' . str_replace(' ', '', ucwords(str_replace(['-', '_'], ' ', $key)));
-
-			if (method_exists($this, $method))
```

Comment:
please don't remove it. It is useful when we want to filter property value during fill via `setProperty($value)`. removing it will be BC break.

Reply:
oh, just realize there is `__set`

Reply:
Yup, that part of the code also exists in __set, is there a need to check whether there is a setter method for each property twice?